guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -12,38 +12,65 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. Ensure the Server and {project-client-name} repositories are enabled.
+** For {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+** For {EL} 7:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+--enable={RepoRHEL7ServerSatelliteUtils}
 ----
 endif::[]
 . Install the Pulp Manifest package:
+** For {EL} 8:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {package-install-project} python3-pulp_manifest
 ----
+ifdef::satellite[]
 +
 Note that this command stops the {Project} service and re-runs {foreman-installer}.
 Alternatively, to prevent downtime caused by stopping the service, you can use the following:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-ifdef::satellite[]
-# subscription-manager repos --enable {RepoRHEL7ServerSatelliteToolsProductVersion}
-endif::[]
 # {foreman-maintain} packages unlock
-# yum install install python-pulp-manifest -y
+# {package-install} python3-pulp_manifest
 # {foreman-maintain} packages lock
-ifdef::satellite[]
-# subscription-manager repos --disable {RepoRHEL7ServerSatelliteToolsProductVersion}
+----
 endif::[]
++
+This installs the package without downtime.
```

Comment:
This must be in the above `ifdef`. Or it can be dropped since on line 43 there's already "to prevent downtime".